### PR TITLE
Experimental Features: FF109 enables 'scrollend' by default

### PR DESCRIPTION
**Pull request by bsmth**
This PR adds experimental note for the `onscrollend` event which is fired when scrolling is complete inside scrollable elements or a Document.

This was added in 108 behind the pref which is enabled by default in 109.

__Related issues and pull requests:__
- [x] BCD https://github.com/mdn/browser-compat-data/pull/18407
- [x] Content https://github.com/mdn/content/pull/22894
- [ ] parent issue https://github.com/mdn/content/issues/22814

### CODE REVIEW

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -1648,6 +1648,49 @@ See these bugs for details: {{bug(1556362)}}, {{bug(1556373)}}, {{bug(1556365)}}
   </tbody>
 </table>
 
+#### Document and element scrollend events
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/experimental_features/index.md:1651]: Element is a proper name
```suggestion
#### Document and Element scrollend events
```

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -1648,6 +1648,49 @@ See these bugs for details: {{bug(1556362)}}, {{bug(1556373)}}, {{bug(1556365)}}
...
+
+The `scrollend` event indicates that the user has completed scrolling in {{domxref("Element")}} and {{domxref("Document")}} objects.
+For more information, see the [Element scrollend event](/en-US/docs/Web/API/Element/scrollend_event) and [Document scrollend event](/en-US/docs/Web/API/Document/scrollend_event) documentation ({{bug(1797013)}}, {{bug(1803435)}}).
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/experimental_features/index.md:1654]:
```suggestion
For more information, see [Element: `scrollend` event](/en-US/docs/Web/API/Element/scrollend_event) and [Document: `scrollend` event](/en-US/docs/Web/API/Document/scrollend_event) ({{bug(1797013)}}, {{bug(1803435)}}).
```

**Review by hamishwillee — APPROVED** [review]
The issue name didn't match the content. I like that you've put this in but of course you'll have to remove it next month when FF109 is near release.

**Comment by bsmth** [comment]
Thanks Hamish

>you'll have to remove it next month when FF109 is near release

I thought this was best for the moment, thanks for the reminder 👍🏻

**Pull request merged**